### ISSUE TITLE
Active Callback not working for Select + Code

### ISSUE THREAD

**Issue by vlthemes** [themeum/kirki#2020]
```php
VLT_Options::add_field( array(
	'type' => 'select',
	'settings' => 'site_gdpr',
	'section' => 'general_site_protection',
	'label' => esc_html__( 'GDPR Compatibility', 'leedo' ),
	'priority' => $priority++,
	'transport' => 'auto',
	'choices' => array(
		'enable' => esc_html__( 'Enable', 'leedo' ),
		'disable' => esc_html__( 'Disable', 'leedo' )
	),
	'default' => 'enable',
) );

VLT_Options::add_field( array(
	'type' => 'code',
	'settings' => 'site_gdpr_content',
	'section' => 'general_site_protection',
	'label' => esc_html__( 'Checkbox Content', 'leedo' ),
	'priority' => $priority++,
	'transport' => 'auto',
	'choices' => array(
		'language' => 'html',
	),
	'default' => '',
	'active_callback' => array(
		array(
			'setting' => 'site_gdpr',
			'operator' => '==',
			'value' => 'enable'
		)
	)
) );

**Comment by vlthemes** [comment]
Also I have error in console log:

http://take.ms/UgPu6
http://take.ms/RIZCj

**Comment by niklasp** [comment]
can be labeled "support" probably #2026

**Comment by aristath** [comment]
@vlthemes apologies for the delay.
Is this still an issue? Are you using the stable version from w.org, or the dev branch?

**Comment by vlthemes** [comment]
I use wp.org version. Issue not fixed.

**Comment by aristath** [comment]
fixed in the dev branch, will be included in 3.0.35 when released.

**Comment by vlthemes** [comment]
Thank you so much :)
